internal/categories/manager.go

```diff
@@ -46,6 +46,33 @@ func (m *Manager) Get(w http.ResponseWriter, r *http.Request) {
 	writeJson(w, fromDBTypeArray(dbCategories))
 }
 
+func (m *Manager) GetCategoryCounts(w http.ResponseWriter, _ *http.Request) {
+	var dtos map[string]CategoryCountDTO
+	dtos = make(map[string]CategoryCountDTO)
+
+	serviceCounts := m.DbClient.GetCategoryServiceCounts()
+	for _, serviceCount := range serviceCounts {
+		var dto CategoryCountDTO
```

Comment:
I think this might be a problem, you could declare it like this to create the empty struct

`dto := CategoryCountDTO{}`

Reply:
Sure thanks